digital-anonymity.md

```diff
@@ -0,0 +1,66 @@
+| Author | TallAmericano |
+| :---- | :---- |
+| Title | Tools and Strategies for Digital Anonymity |
```

Comment:
Better title needed. Once you've written your thesis statement we should be able to come up with a better one.

Reply:
Just a heads up, for your next article, I've edited the "Article Template" which now includes more detail to help with the planing stage of an article and the social media stage of the article.

Reply:
Sorry I had just put this template up without properly updating a few parts. I'll also look at the new article template going forward

I've added thesis: The piece aims to look into the world of cypherpunk tools, strategies, and technologies designed to protect individuals digital anonymity in the face of increasing surveillance.